setup_cfg_fmt.py

```diff
@@ -100,6 +100,24 @@ def cb(match: Match[str]) -> str:
     return GLOB_PART.sub(cb, s)
 
 
+def _first_valid_readme_file(setup_cfg: str) -> str | None:
+    prefix = _case_insensitive_glob('readme')
+    path = _adjacent_filename(setup_cfg, prefix)
+    readme_files = [
+        filename for filename in sorted(glob.iglob(f'{path}*'))
+        if os.path.isfile(filename)
+    ]
+    for filename in readme_files:
+        if (
+            not filename.endswith(('.adoc', 'asciidoc'))
```

Comment:
I think this entire function can be replaced with a sort key which puts asciidoc last -- it can be a small 1-2 line modification on the existing function rather than copy pasting